tests/testsuite/git_status.rs

```diff
@@ -259,33 +259,54 @@ fn doesnt_show_untracked_file_if_disabled() -> io::Result<()> {
 }
 
 #[test]
-#[ignore]
```

Comment:
Keep the `[ignore]` and add to new tests according to this comment: https://github.com/starship/starship/pull/434#discussion_r329673883

These tests can be run specifically with `cargo test git_status -- --ignored`

Reply:
Updated with `#[ignore]` back in place.

For the new tests, were there any edge cases you'd like me to explore further than what I did cover?

Reply:
Tests look good 👍🏼